SimPEG/Examples/PF_Gravity_Laguna_del_Maule_Inversion.py

```diff
@@ -1,246 +0,0 @@
-# coding: utf-8
```

Comment:
I don't think we want to remove this... It already is on dev. To fix, you can just copy the one from dev and bring it back here.